ethereum_history_api/oracles/src/util/hex.ts

```diff
@@ -0,0 +1,30 @@
+import { assert } from './assert.js';
+import { BYTE_HEX_LEN } from './const.js';
+
+const PREFIX = '0x';
+
+export function hasPrefix(hex: string): boolean {
```

Comment:
Should it be:
`hasHexPrefix`
`addHexPrefix`
`removeHexPrefix` or `trimHexPrefix`

Reply:
This is within `hex.ts` file so we can omit this, no?